src/ray/core_worker/core_worker.h

```diff
@@ -138,6 +138,9 @@ class TaskCounter {
   // Used for actor state tracking.
   std::string actor_name_ ABSL_GUARDED_BY(mu_);
   int64_t num_tasks_running_ ABSL_GUARDED_BY(mu_) = 0;
+
+  // Metrics
+  ray::observability::MetricInterface &metric_tasks_;
```

Comment:
what does `metric_tasks_` mean? let's name it something more clear like `task_counter_` (I don't know what it actually is)